R/remove_empty.R

```diff
@@ -42,11 +44,19 @@
 #' # remove empty columns and rows
 #' remove_empty(tmp)
 #' @export
-empty_columns <- function(x) {
+empty_columns <- function(x, include_empty_string = TRUE) {
   if ((!is.matrix(x) && !is.data.frame(x)) || ncol(x) < 2) {
     vector("numeric")
   } else {
-    which(colSums(is.na(x)) == nrow(x))
+    all_na <- colSums(is.na(x)) == nrow(x)
+    if (include_empty_string) {
+      all_empty <- sapply(x, function(i) {
```

Comment:
you can use `vapply()` instead to save a few microseconds 😄

Reply:
oh yes, `vapply` and `tapply` receive too little love from R coders ;-)